src/modules/epsr/process.cpp

```diff
@@ -124,12 +126,8 @@ bool EPSRModule::process(Dissolve &dissolve, const ProcessPool &procPool)
     const auto mcoeff = 200;
 
     // Calculate some values if they were not provided
-    if (rMaxPT_ < 0.0)
-        rMaxPT_ = dissolve.pairPotentialRange();
-    if (rMinPT_ < 0.0)
-        rMinPT_ = rMaxPT_ - 2.0;
-    if (!nCoeffP_)
-        nCoeffP_ = std::min(int(10.0 * rMaxPT_ + 0.0001), mcoeff);
+    auto rmaxpt = rMaxPT_ ? rMaxPT_.value() : dissolve.pairPotentialRange();
+    auto rminpt = rMinPT_ ? rMinPT_.value() : rmaxpt - 2.0;
```

Comment:
Changing this bit also removed the setting of `nCoeffP_` optional which is now causing a crash. We need to do a similar thing as above, and change uses of `nCoeffP_.value()` to use our temporary var:
```suggestion
    auto rminpt = rMinPT_ ? rMinPT_.value() : rmaxpt - 2.0;
    auto ncoeffp = nCoeffP_ ? nCoeffP_.value() : std::min(int(10.0 * rmaxpt + 0.0001), mcoeff);
```